content/en/docs/setup/learning-environment/kind.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Installing Kubernetes with Kind
+weight: 20
```

Comment:
Because the Minikube page has `weight: 30`, I suggest revising this to:
```suggestion
weight: 40
```

Reply:
Updated.